modules/ROOT/pages/provisioning-exoscale.adoc

```diff
@@ -0,0 +1,64 @@
+= Provisioning Fedora CoreOS on Exoscale
+
+This guide shows how to provision new Fedora CoreOS (FCOS) instances on https://exoscale.com[Exoscale] Cloud Hosting.
+
+== Prerequisites
+
+Before provisioning a FCOS instance, you can use or not an Ignition configuration file containing your customizations. If you need one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You also need to have access to an Exoscale account. https://portal.exoscale.com/register[Register] if you don't have one.
+
+== About images
+
+Fedora CoreOS is updated at every major change on our platform. The future goal is to have Exoscale FCOS in https://getfedora.org/en/coreos/download?tab=cloud_launchable[Cloud Lauchable] section, to be updated at every release.
+
+== Launching a VM instance
+
+You can provision a FCOS instance using the Exoscale https://portal.exoscale.com/compute/instances/add?os-group=coreos[Web Portal]. Then you can choose your FCOS version and follow the instance creation wizard.
+
+If you just want SSH access and no further customization, you don't need to pass any custom instance user-data.
+Depending on your Exoscale Organization, relevant SSH public keys will be automatically added to the VM. This provides an easy way to test out FCOS without first creating an Ignition config.
+
+If you need an Ignition config, paste-it in the User Data section in the instance creation wizard.
+
+Otherwise, you can provision a FCOS instance using https://community.exoscale.com/documentation/tools/exoscale-command-line-interface/[Exoscale CLI]
+
+.Launching a new instance with Exoscale CLI
+[source, bash]
+----
+VM_NAME='fcos-node01'
+exo vm create "${VM_NAME}" \
+    --template "Linux Fedora CoreOS 32 64-bit" \
+    --keypair "my_key_name" \
+    --cloud-init-file "path/to/ignition-file.ign" # Optional, if you want to launch a customized FCOS instance.
+----
+
+.SSH to the instance
+[source, bash]
+----
+VM_NAME='fcos-node01'
+exo ssh "${VM_NAME}"
+----
+
+== Launching a VM instance as a https://community.exoscale.com/documentation/compute/custom-templates[custom Template]
+
+Download the Qcow2 image on the Fedora's website in the section https://getfedora.org/en/coreos/download?tab=cloud_operators[Cloud Operators].
+
+
+.Unxz the file
+[source, bash]
+----
+unxz "fedora-coreos-$VERSION-exoscale.x86_64.qcow2.xz"
+----
+
+You need to resize the image virtual size to 10G.
+
+.Resize image virtual size with qemu-img
+[source, bash]
+----
+qemu-img resize "fedora-coreos-$VERSION-exoscale.x86_64.qcow2" +2G
```

Comment:
Is this necessary?  Does exoscale not support specifying the root device size at provisioning time?  Why the `+2G`?

Reply:
@pierre-emmanuelJ - it might be worth you adding a comment to https://github.com/coreos/fedora-coreos-tracker/issues/492 - depending on how we implement that it could be applied more generically.

Reply:
even better if you had some cycles and wanted to try to drive that feature within COSA